index.js

```diff
@@ -74,8 +73,12 @@ class DHT extends EventEmitter {
   static bootstrapper (port, host, opts) {
     if (!port) throw new Error('Port is required')
     if (!host) throw new Error('Host is required')
-    const id = peer.id(host, port)
```

Comment:
No problem in calculating the id here for the Table in the constructor, but removing `opts.id` makes the bootstrap node more equal to any other node in terms of options which I think it's good

The flow without `opts.id` is as follows:

Random table ID in the constructor:
https://github.com/mafintosh/dht-rpc/blob/1e091ff76b6b0607d6b1c01dcb00f434b38cecec/index.js#L30

Later, as part of the initial `_bootstrap` process, this `_updateNetworkState` gets executed:
https://github.com/mafintosh/dht-rpc/blob/1e091ff76b6b0607d6b1c01dcb00f434b38cecec/index.js#L584

Then this will not match, because the constructor used a random ID and the id above will differ:
https://github.com/mafintosh/dht-rpc/blob/1e091ff76b6b0607d6b1c01dcb00f434b38cecec/index.js#L607

So the table gets re-created:
https://github.com/mafintosh/dht-rpc/blob/1e091ff76b6b0607d6b1c01dcb00f434b38cecec/index.js#L610

That could have been saved by having `opts.id`, but it's better to have the Table id generation in one place as right now in `_updateNetworkState`